examples/recipe-sourcing-contentful/src/pages/index.js

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { Link } from "gatsby"
+
+import Layout from "../components/layout"
+import Image from "../components/image"
+import SEO from "../components/seo"
+
+const IndexPage = () => (
+  <Layout>
+    <SEO title="Home" />
+    <h1>Hi people</h1>
+    <Link to="/news">View example news page</Link>
```

Comment:
Can we put the News stuff on the index page, and simplify this one a bit?

Reply:
Added in https://github.com/gatsbyjs/gatsby/pull/17318/commits/6bf620110f66fc4a9f9a982c7c38f32d166316b2